django/db/models/base.py

```diff
@@ -611,11 +611,7 @@ def refresh_from_db(self, using=None, fields=None):
             setattr(self, field.attname, getattr(db_instance, field.attname))
             # Throw away stale foreign key references.
```

Comment:
Perhaps the comment should say `Clear cached foreign keys.` since the part about "stale" may no longer be true.